airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -77,7 +77,7 @@
 
   private static final long GRACEFUL_SHUTDOWN_MINUTES = 5L;
   private static final int MIN_RECORDS = 500;
-  private static final int BATCH_SIZE = 500;
+  private static final int BATCH_SIZE = 10000;
```

Comment:
@cgardens We need to make sure it doesn't break the other destinations.

Snowflake had a hard limit on the batch size